cli/README.md

```diff
@@ -91,15 +91,14 @@ path|status|description
 [jobs/single-step/scikit-learn/iris/job-docker-context.yml](jobs/single-step/scikit-learn/iris/job-docker-context.yml)|[![jobs/single-step/scikit-learn/iris/job-docker-context](https://github.com/Azure/azureml-examples/workflows/cli-jobs-single-step-scikit-learn-iris-job-docker-context/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/cli-jobs-single-step-scikit-learn-iris-job-docker-context.yml)|Train a scikit-learn SVM on the Iris dataset using a custom Docker container build.
 [jobs/single-step/scikit-learn/iris/job-sweep.yml](jobs/single-step/scikit-learn/iris/job-sweep.yml)|[![jobs/single-step/scikit-learn/iris/job-sweep](https://github.com/Azure/azureml-examples/workflows/cli-jobs-single-step-scikit-learn-iris-job-sweep/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/cli-jobs-single-step-scikit-learn-iris-job-sweep.yml)|Sweep hyperparemeters for training a scikit-learn SVM on the Iris dataset.
 [jobs/single-step/scikit-learn/iris/job.yml](jobs/single-step/scikit-learn/iris/job.yml)|[![jobs/single-step/scikit-learn/iris/job](https://github.com/Azure/azureml-examples/workflows/cli-jobs-single-step-scikit-learn-iris-job/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/cli-jobs-single-step-scikit-learn-iris-job.yml)|Train a scikit-learn SVM on the Iris dataset.
-[jobs/single-step/spark/nyctaxi/job.yml](jobs/single-step/spark/nyctaxi/job.yml)|[![jobs/single-step/spark/nyctaxi/job](https://github.com/Azure/azureml-examples/workflows/cli-jobs-single-step-spark-nyctaxi-job/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/cli-jobs-single-step-spark-nyctaxi-job.yml)|This sample shows how to run a single node Spark job on Azure ML. The 47GB NYC Taxi dataset is read in parquet format by a 1 node Spark cluster, processed and then written as job output in parquet format.
```

Comment:
why all the other changes?

Reply:
Followed the [contribution instructions](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md#set-up-and-pre-pr) and ran `cli/readme.py`. These changes came from it.